src/index.ts

```diff
@@ -17,15 +17,20 @@
 
 import * as tfc from '@tensorflow/tfjs-core';
 
-import {nodeFileSystemRouter} from './io/file_system';
-import {NodeJSKernelBackend} from './nodejs_kernel_backend';
+import { nodeFileSystemRouter } from './io/file_system';
+import { NodeJSKernelBackend } from './nodejs_kernel_backend';
 
 // tslint:disable-next-line:no-require-imports
 import bindings = require('bindings');
-import {TFJSBinding} from './tfjs_binding';
+import { TFJSBinding } from './tfjs_binding';
```

Comment:
Hrm - did you use VSCode w/ the clang extension? This looks wrong, see https://github.com/tensorflow/tfjs-node/blob/master/src/nodejs_kernel_backend.ts#L19

Reply:
You are right, I can not log into my desktop machine and I have to reinstall goobuntu, but forgot to install clang extension.